kiwi/xml_state.py

```diff
@@ -1243,7 +1245,8 @@ def get_volumes(self):  # noqa C901
                 'mountpoint',
                 'fullsize',
                 'label',
-                'attributes'
+                'attributes',
+                'is_root_volume'
```

Comment:
nit: the is_root_volume tuple attribute is not included in the above docs comment.

Reply:
yep, will fix it :)

Reply:
done